README.rst

```diff
@@ -38,74 +38,149 @@ This repository uses `git-flow <https://github.com/nvie/gitflow>`_ conventions;
 contains the most recent release, and work in progress will be on the **develop** branch.
 Pull requests should be made against **develop**.
 
-
-Development instructions
-------------------------
-
-Bare-metal
-~~~~~~~~~~
-
-Initial setup and installation:
-
-- Recommended: create and activate a python 3.9 virtualenv::
-
-    virtualenv cdhweb -p python3.9
-    source cdhweb/bin/activate
-
-- Use pip to install required python dependencies.  To install production
-  dependencies only::
-
-    pip install -r requirements.txt
-
-  To install all development requirements::
-
-    pip install -r requirements/dev.txt
-
-- Install django-compressor dependencies::
-
-    npm install
-
-- Copy sample local settings and configure for your environment::
-
-    cp cdhweb/settings/local_settings.py.sample cdhweb/settings/local_settings.py
-
-You must add a ``SECRET_KEY`` value in your local settings.
-
-- Download licensed fonts and install locally under /sitemedia/fonts/
-
-- Install OpenCV dependencies (if necessary) for `wagtail image feature detection <https://docs.wagtail.io/en/stable/advanced_topics/images/feature_detection.html>`_
-
-Docker (Springload)
+-----------------
+Development Setup
+-----------------
+
+Choose one of the following setup options:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Option 1: Local Development ("bare-metal")
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This option runs the application directly on your local machine.
+
+**Setup Steps:**
+
+1. **Set up Python environment:**
+   .. code-block:: bash
+      # Install Python 3.11 (if not already installed)
+      brew install python@3.11  # macOS
+      
+      # Create and activate virtual environment
+      virtualenv cdhweb -p python3.11
+      source cdhweb/bin/activate
+
+. **Install Python dependencies:**
+   .. code-block:: bash
+      # For development (includes test dependencies)
+      pip install -r requirements/dev.txt
+      
+      # Or for production only
+      pip install -r requirements.txt
+
+4. **Set up local settings:**
+   .. code-block:: bash
+      cp cdhweb/settings/local_settings.py.sample cdhweb/settings/local_settings.py
+      
+      # Edit local_settings.py and add your SECRET_KEY
+      # Configure database connection to your local PostgreSQL
+
+5. **Install Node.js dependencies:**
+   .. code-block:: bash
+      npm install
+
+6. **Set up database:**
+   .. code-block:: bash
+      # Create database
+      createdb cdhweb
+      
+      # Run migrations
+      python manage.py migrate
+      
+      # Create superuser (optional)
+      python manage.py createsuperuser
```

Comment:
It's also possible to initialize a princeton netid account with admin permissions ; both of these accounts may be useful.

```
python manage.py createcasuser --admin netid
```